cmd/deploy.go

```diff
@@ -52,16 +52,17 @@ func newDeployCmd(rootConfig *RootCommandConfig) *cobra.Command {
 		Short: "Build and deploy your Appsody project to Kubernetes.",
 		Long: `Build and deploy a local container image of your Appsody project to your Kubernetes cluster. 
 		
-The command extracts the stack, along with your Appsody project to a local directory. The extracted project is used to build the container image for deployment using 'appsody build'.
-A deployment manifest file, "app-deploy.yaml", is generated if one is not present, and applied to the cluster. The CLI uses this file to deploy your image to your Kubernetes cluster via the Appsody Operator, and optionally as a Knative service if you specify the "--knative" flag.
+The command performs the following steps:
 
-
-If the Appsody Operator cannot be found, it will be installed on your cluster.`,
+1. Extracts the stack, along with your Appsody project, to a local directory.
+2. Runs the appsody build command to build the container image for deployment.
+3. Generates a deployment manifest file, "app-deploy.yaml", if one is not present, then applies it to your Kubernetes cluster.
+4. Deploys your image to your Kubernetes cluster via the Appsody operator, or optionally as a Knative service if you specify the "--knative" flag. If the Appsody Operator cannot be found, it will be installed on your cluster.`,
```

Comment:
Need lowercase "o" for operator. I don't think we need "optionally" now that we know it's a mutually exclusive thing. One more minor thing - is it "the" operator, or "an" operator? The docs suggest you can have multiple operators?
"4. Deploys your image to your Kubernetes cluster via the Appsody operator, or as a Knative service if you specify the "--knative" flag. If the Appsody operator cannot be found, it will be installed on your cluster."

Reply:
You can have multiple operators. I've changed this now